okteto/keepalive.go

```diff
@@ -0,0 +1,128 @@
+package okteto
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc/keepalive"
+)
+
+var (
+	kcp *keepalive.ClientParameters
```

Comment:
Why do we have this as global variables? I don't think it's necessary, we can skip the `ip k__ == nil` in all the functions. 

Also, if you are interested you can checkout the Functional Options Pattern at https://golang.cafe/blog/golang-functional-options-pattern.html (Option 3)

Reply:
Im using these vars to avoid read envs again in case we need to add keepalive params in other clients/servers. A singleton pattern

Reply:
Sounds good 👍 
In the future I think we should have an interface if we want to use the singleton pattern, so that we have limited ways to interact with the singleton. As it is now, one not knowing how to handle the singleton could misuse it. And probably it would need a mutex lock.